Unable to process Policy Set JSON files in folder './definitions/policySetDefinitions': Confirm-ObjectValueEqualityDeep error

**Describe the bug**
A clear and concise description of what the bug is.
after upgrading 

**To Reproduce**
Steps to reproduce the behavior:
Build policy plan: 
```PowerShell
Build-DeploymentPlans -PacEnvironmentSelector ${{ parameters.pacEnvironmentSelector }} -DevOpsType "ado" -InformationAction Continue
```
Error:

```PowerShell 
Processing Policy Set JSON files in folder './definitions/policySetDefinitions'
===============================================================================================
Number of Policy Set files = 50
Confirm-ObjectValueEqualityDeep: /home/vsts/.local/share/powershell/Modules/EnterprisePolicyAsCode/10.7.7/internal/functions/Confirm-ObjectValueEqualityDeep.ps1:178
Line |
 178 |  …                $match = Confirm-ObjectValueEqualityDeep $item1 $item2
     |                            ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
     | A parameter cannot be found that matches parameter name 'InputObject'.

##[error]PowerShell exited with code '1'.
```

**Expected behavior**
- Successful processing of all policy set definition in folder './definitions/policySetDefinitions' 
- Successful creation of a policy plan ready to deploy

**Screenshots**
If applicable, add screenshots to help explain your problem.

**EPAC Version**
- EnterprisePolicyAsCode : 10.7.7
- PowerShell : 7.4.7
- Az version: 12.1.0

--- a/Scripts/Helpers/Confirm-ObjectValueEqualityDeep.ps1
+++ b/Scripts/Helpers/Confirm-ObjectValueEqualityDeep.ps1
@@ -40,7 +40,7 @@ function Confirm-ObjectValueEqualityDeep {
                 # This resolves a unique situation where GraphApi returns a string instead of an array for a metadata value, very unique situation
                 # By testing the json and count, we can confirm that it is a json string and convert it to an array
                 # A try catch is included to revert to the value we normally use in the event of failure just in case there is a scenario that has not been accounted for
-                if ($(Test-Json -InputObject $Object1) -and $Object1.count -eq 1) {
+                if ($(Test-Json -Json $Object1) -and $Object1.count -eq 1) {
                     try {
                         $Object1 = $Object1 | ConvertFrom-Json -Depth 100
                     }
@@ -54,7 +54,7 @@ function Confirm-ObjectValueEqualityDeep {
                 # This resolves a unique situation where GraphApi returns a string instead of an array for a metadata value, very unique situation
                 # By testing the json and count, we can confirm that it is a json string and convert it to an array
                 # A try catch is included to revert to the value we normally use in the event of failure just in case there is a scenario that has not been accounted for
-                if ($(Test-Json -InputObject $Object2) -and $Object2.count -eq 1) {
+                if ($(Test-Json -Json $Object2) -and $Object2.count -eq 1) {
                     try {
                         $Object2 = $Object2 | ConvertFrom-Json -Depth 100
                     }

